tester_app/client/app/bundles/HelloWorld/startup/clientGlobals.jsx

```diff
@@ -0,0 +1,4 @@
+import HelloWorldAppClient from './HelloWorldAppClient';
+
+// This is how react_on_rails can see the HelloWorldApp in the browser.
+window.HelloWorldApp = HelloWorldAppClient;
```

Comment:
See above:

@alexfedoseev should keep the Client in the file/component name when ONLY during client rendering?